default.nix

```diff
@@ -221,6 +225,25 @@ let
       };
     };
 
+    plutus-contract-exe = rec {
+      static = pkgs.haskell.lib.justStaticExecutables;
```

Comment:
I don't think it should be necessary to make things static - they should just work in the docker container.

Reply:
I tried it without `static` but it massively blew up the image size. static: 16 MB. non-static: 3.478 GB.

Reply:
OK, 588 MB compressed (see https://hub.docker.com/r/jannmueller/plutus-contract-exe/tags)

But still, I think having a tiny image is worth the `justStaticExecutables`

Reply:
Huh, interesting. Worth a comment, at least!